tests/test.py

```diff
@@ -0,0 +1,100 @@
+import glob
+import os
+import re
+import subprocess
+import sys
+import time
+import shutil
+import socket
+
+from avocado import VERSION
+from avocado import Test
+
+class MinishiftISOTest(Test):
+
+    def setUp(self):
+        ''' Test Setup '''
+        self.log.info("################################################################")
+        self.log.info("Avocado version : %s" % VERSION)
+        self.log.info("################################################################")
+
+        self.repo_dir = os.path.dirname(os.path.realpath(__file__)) + "/.."
+        self.scripts_dir = self.repo_dir + "/tests/"
+        self.bin_dir = self.repo_dir + "/bin/"
+        self.docker_machine_vm = "docker-machine-test-vm"
+        self.iso_name = 'minishift-b2d'
+
+        # Find iso files and skip test if no ISO file is found
+        self.iso_file = self.repo_dir + "/build/%s.iso" % self.iso_name
+        if not os.path.isfile(self.iso_file):
+            self.skip("Skipping testing as no ISO found in 'build' directory.")
+
+    def test_boot_vm_out_of_iso(self):
+        ''' Test booting up VM out of ISO '''
+        cmd = self.bin_dir + "docker-machine create %s -d kvm --kvm-boot2docker-url=%s" % (self.docker_machine_vm, self.iso_file)
```

Comment:
I am confused now. I thought we are adding minishift smoke tests. This is docker-machine, right? So at best we can verify that VM creation works. Fair enough, but for this particular repo I am more interested that this works with the latest version of minishift.